create_cluster.py

```diff
@@ -576,6 +578,9 @@ def cli(cluster_name: str, regions: list, cluster_size: int, instance_type: str,
     if not re.match(cluster_name_re, cluster_name):
         raise click.UsageError('Cluster name must only contain lowercase latin letters, digits and dashes (it also must start with a letter and cannot end with a dash), in other words it must be matched by the following regular expression: {}'.format(cluster_name_re))
 
+    if not application_name:
+        raise.click.UsageError('Please specify application name which shell be used.')
```

Comment:
Also, I believe the cluster-name is already used as a unique KIO application_id. Can't this be the the AD application name? It's already creepy the amount of parameters we have